queue.rb

```diff
@@ -21,6 +24,8 @@
 
 scheduler = Rufus::Scheduler.new
 
+PINS_TO_KEEP = 3
```

Comment:
Shouldn’t you keep all pins that are still being played?

Reply:
This constant ensures the latest pins are kept even if they're not used, otherwise all unused pins are discarded. I've added a comment and updated the constant name to clarify this.